CHANGES.md

```diff
@@ -140,6 +140,11 @@
 
 ### Bug Fixes and Minor Changes
 
+  * `XMonad.Layout.Fullscreen`
+
+    Add fullscreenSupportBorder which uses smartBorders to remove window borders
+    when the window is fullscreen.
```

Comment:
In accordance with the rest of the (now thankfully uniform) section, please use a bullet point here, e.g.

```suggestion
    - Add fullscreenSupportBorder which uses smartBorders to remove
      window borders when the window is fullscreen.
```